### DISCUSSION TITLE
other language

### DISCUSSION THREAD

**Discussion by foXaCe** [BottlecapDave/HomeAssistant-OctopusEnergy#489] [General]
hello,

i use the french version of the web page

the developper link do not work

do you have other link for the api please ?

**Comment by BottlecapDave** [comment]
Hi. If it's anything like OE in other terratories, they are Octopus Energy in name but have different infrastructure. This integration has been developed against the UK APIs, but should work in other terratories once they become available. However, this will rely on community assistance.

**Comment by foXaCe** [comment]
yes is the same 
except that there is no developer access

https://www.octopusenergy.fr/

**Reply by BottlecapDave** [reply]
In the UK site, it's available under personal details. I assume it would be in the same area if it's available.
